src/glue/resources/table_columns.yaml

```diff
@@ -26,7 +26,7 @@ tables:
       - Name: EnrollmentDate
         Type: string
       - Name: CustomFields
-        Type: struct<EhrConnected:string,EhrConnectedDate:string,SymptomLogSetupDate:string,DevicesConnected:string,AppleHealthRecordsReceived:string,DeviceOrderStatus:string,DeviceOrderDate:string,InfectionFirstReportedDate:string,ForceSendInvitation:string,ForceSync:string,SelfEnrolled:string,AppDownloadDate:string,DevicesConnectedDate:string,HasOutstandingSurveys:string,ProjectCode:string,Site:string,DateOfBirthVerified:string,DeviceEligible:string,InfectionsReported:string,AppDownloaded:string,AppleHealthEnabled:string,AppleHealthRecordsEnabled:string,GoogleFitEnabled:string,Reminder1Enabled:string,Reminder2Enabled:string,ReminderTime1:string,ReminderTime2:string,SkipConsent:string,Symptoms:array<struct<id:string,name:string,color:string,severityTracking:string,inactive:boolean>>,Treatments:array<struct<id:string,name:string,color:string,inactive:boolean,EOPReason:int,EOPRemoveData:int>>>
+        Type: struct<EhrConnected:string,EhrConnectedDate:string,SymptomLogSetupDate:string,DevicesConnected:string,AppleHealthRecordsReceived:string,DeviceOrderStatus:string,DeviceOrderDate:string,InfectionFirstReportedDate:string,ForceSendInvitation:string,ForceSync:string,SelfEnrolled:string,AppDownloadDate:string,DevicesConnectedDate:string,HasOutstandingSurveys:string,ProjectCode:string,Site:string,DateOfBirthVerified:string,DeviceEligible:string,InfectionsReported:string,AppDownloaded:string,AppleHealthEnabled:string,AppleHealthRecordsEnabled:string,GoogleFitEnabled:string,Reminder1Enabled:string,Reminder2Enabled:string,ReminderTime1:string,ReminderTime2:string,SkipConsent:string,EOPReason:int,EOPRemoveData:int,EOPDate:string,Symptoms:array<struct<id:string,name:string,color:string,severityTracking:string,inactive:boolean>>,Treatments:array<struct<id:string,name:string,color:string,inactive:boolean>>>
```

Comment:
Apologies, could you explain why EOPRemoveData and EOPRReason didn't show up previously? Is it because it was part of "Treatments" array?

Reply:
@thomasyu888 that's correct.

Reply:
Thanks @philerooski sorry I missed that in the last review!